kustomize/go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/xlab/treeprint v1.2.0 // indirect
 	go.starlark.net v0.0.0-20200306205701-8dd3e2ee1dd5 // indirect
+	golang.org/x/exp v0.0.0-20231006140011-7918f672742d // indirect
```

Comment:
I'm still pretty new to go. Can you please help me understand why this package needed to be added for this change?

Reply:
That's because I was experimenting with `slices.IndexFunc()` and changed my mind later, but forgot to run `go mod tidy` 😅

Thanks for flagging, I just fixed it.